ramses/auth.py

```diff
@@ -193,13 +193,17 @@ def create_system_user(config):
         s_user = settings['system.user']
         s_pass = settings['system.password']
         s_email = settings['system.email']
+        defaults = dict(
+            password=s_pass,
+            email=s_email,
+            groups=['admin'],
+        )
+        if config.registry.database_acls:
+            from pyramid.security import Allow, ALL_PERMISSIONS
```

Comment:
Not sure why this import is here rather than at the top of the module. I don't see any advantage to delaying this import. It's going to be imported anyway in other parts of the application.